api/routers/edr.py

```diff
@@ -267,7 +244,17 @@ async def get_data_position(
             detail={"coords": f"Unexpected error occurred during wkt parsing: {coords}"},
         )
 
-    return await get_data_area(coords=poly.wkt, parameter_name=parameter_name, datetime=datetime, f=f)
+    request = dstore.GetObsRequest(
+        # 10 meters around the point
+        spatial_circle=dstore.Circle(center=dstore.Point(lat=point.y, lon=point.x), radius=0.01),
```

Comment:
0.01 is approx 1.1 km? I think radius should be 1e-4 if this radius is equivalent to decimal degrees? Ref. https://en.wikipedia.org/wiki/Decimal_degrees

Reply:
Radius for this gRPC parameter is in kilometers:
https://github.com/EURODEO/e-soh/blob/main/protobuf/datastore.proto#L56